ci/provisioning/post_provision_config.sh

```diff
@@ -12,13 +12,53 @@ host wolf-*
     LogLevel error
 EOF
 
+: "${DAOS_STACK_RETRY_DELAY_SECONDS:=60}"
+: "${DAOS_STACK_RETRY_COUNT:=3}"
+: "${BUILD_URL:=Not_in_jenkins}"
+: "${OPERATIONS_EMAIL:=$USER@localhost}"
+
+retry_cmd() {
+    local command="$1"
+    shift
+
+    local tries=$DAOS_STACK_RETRY_COUNT
+    while [ $tries -gt 0 ]; do
+        if time $command "$@"; then
+            # succeeded, return with success
+            return 0
+        fi
+        # We hit an error
+        (( tries-- ))
+        {
+          echo "Command $command $* failed on $HOSTNAME for $BUILD_URL"
+          echo "Command status was ${PIPESTATUS[0]}"
+          echo "Will retry $tries before giving up."
+        } 2>&1 | mail -s "Command failed in $BUILD_URL" \
+                      -r "$HOSTNAME"@intel.com "$OPERATIONS_EMAIL"
+
+        sleep "$DAOS_STACK_RETRY_DELAY_SECONDS"
```

Comment:
To avoid an unnecessary sleep, we should not execute this line if `$tries` is `0`.

Reply:
Changed in the next push, also removed the useless continue statement.